polkadot-protocol/smart-contracts-basics/overview.md

```diff
@@ -0,0 +1,152 @@
+
+[Ethereum addresses (ECDSA)](https://ethereum.org/en/glossary/#address){target=\_blank} can also be mapped directly to and from the Polkadot SDK's SS58 scheme from existing accounts. Moreover, you can modify Polkadot SDK to use the ECDSA signature scheme directly to avoid any mapping.
+
+At a high level, [Frontier](https://github.com/polkadot-evm/frontier){target=\_blank} is composed of three main components:
+
+- [**Ethereum Client**](https://github.com/polkadot-evm/frontier/tree/master/client){target=\_blank} - an Ethereum JSON RPC compliant client that allows any request coming from an Ethereum tool, such as [Remix](https://remix.ethereum.org/){target=\_blank}, [Hardhat](https://hardhat.org/){target=\_blank} or [Foundry](https://getfoundry.sh/){target=\_blank}, to be admitted by the network
+- [**Pallet Ethereum**](https://docs.rs/pallet-ethereum/latest/pallet_ethereum/){target=\_blank} - a block emulation and Ethereum transaction validation layer that works jointly with the Ethereum client to ensure compatibility with Ethereum tools
+- [**Pallet EVM**](https://docs.rs/pallet-evm/latest/pallet_evm/){target=\_blank} - access layer to the [Rust-based EVM](https://github.com/rust-ethereum/evm){target=\_blank}, enabling the execution of EVM smart contract logic natively
+
+Broadly speaking, in this configuration, an EVM transaction follows the path presented in the diagram below:
+
+``` mermaid
+flowchart TD
+    A[Users and Devs] -->|Send Tx| B[Frontier RPC Ext]
+    subgraph C[Pallet Ethereum]
+        D[Validate Tx]
+        E[Send<br/>Valid Tx]    
+    end
+    B -->|Interact with| C
+    D --> E
+    subgraph F[Pallet EVM]
+        G[Rust EVM]
+    end
+    I[(Current EVM<br/>Emulated State)]
+
+    H[Smart Contract<br/>Solidity, Vyper...] <-->|Compiled to EVM<br/>Bytecode| I
+
+    C --> F
+    I --> F
+    F --> J[(New Ethereum<br/>Emulated State)]
+    F --> K[Execution Logs]
+
+    style C fill:#ffffff,stroke:#000000,stroke-width:1px
+    style F fill:#ffffff,stroke:#000000,stroke-width:1px
+```
+
+Although it seems complex, users and developers are abstracted of that complexity, and tools can easily interact with the parachain as they would with any other EVM-compatible environment.
+
+The Rust EVM is capable of executing regular [EVM bytecode](https://www.ethervm.io/){target=\_blank}. Consequently, any language that compiles to EVM bytecode can be used to create programs that the parachain can execute.
+
+You can find more information on deploying EVM smart contracts to [Polkadot's native smart contract platform](/develop/smart-contracts/evm/native-evm-contracts/){target=\_blank}, or any of [the ecosystem parachains](/develop/smart-contracts/evm/parachain-contracts/){target=\_blank}.
```

Comment:
This is strange, the section should not mention PolkaVM contracts, just Parachain ones. Also, I think the parachain-contracts page will be removed so I am not sure if we need to link there